tutorials/docs-00-getting-started/00_getting-started.jmd

```diff
@@ -0,0 +1,78 @@
+---
+title: Getting Started
+permalink: /:collection/:name/
+redirect_from: docs/0-gettingstarted/
+weave_options:
+  error : false
+---
+
+# Getting Started
+
+## Installation
+
+To use Turing, you need to install Julia first and then install Turing.
+
+### Install Julia
+
+You will need to install Julia 1.3 or greater, which you can get from [the official Julia website](http://julialang.org/downloads/).
+
+### Install Turing.jl
+
+Turing is an officially registered Julia package, so you can install a stable version of Turing by running the following in the Julia REPL:
+
+```julia
+using Pkg
+Pkg.add("Turing")
+```
+
+You can check if all tests pass by running (it might take a long time)
+
+```{julia; eval=false}
+Pkg.test("Turing")
+```
+
+## Example
+
+Here's a simple example showing Turing in action.
+
+First, we can load the Turing and StatsPlots modules
+
+```julia
+using Turing
+using StatsPlots
+```
+
+Then, we define a simple Normal model with unknown mean and variance
+
+```julia
+@model function gdemo(x, y)
+    s² ~ InverseGamma(2, 3)
+    m ~ Normal(0, sqrt(s²))
+    x ~ Normal(m, sqrt(s²))
+    return y ~ Normal(m, sqrt(s²))
+end
+```
+
+Then we can run a sampler to collect results. In this case, it is a Hamiltonian Monte Carlo sampler
+
+```julia
+chn = sample(gdemo(1.5, 2), HMC(0.1, 5), 1000)
+```
+
+We can use `describe` to summarise results
+
+```julia
+describe(chn)
+```
```

Comment:
Better to remove this since the desired summary statistics are shown as output to the sample call above but not when calling `describe` here:

```suggestion
```

Actually I fixed this in quite a few tutorials recently.

Reply:
Maybe another possible improvement might be to rather compare the results and plots with the exact posterior distribution instead of showing `savefig` which is more a Plots.jl detail.

Reply:
It sounds like an interesting exercise for Bayesian inference 101. Can you work this out? @cgbotta

Reply:
I think that is a great idea. I changed a decent bit of the tutorial so that the format is a bit easier to follow for beginners (in my opinion) and has the user generate data so that they can observe the model learning the correct parameters. 

Let me know your thoughts!

Reply:
Ah I misinterpreted - apologies. I can move forward with this approach 👍

Reply:
Okay this has been a good exercise, but I do have a question since I am still learning. 

If I understand correctly, having an unknown mean and variance results in having a prior with 4 parameters to update using the below equations. 

<img width="1108" alt="image" src="https://user-images.githubusercontent.com/39240537/197305326-99ea7913-9fa3-4d2e-a702-36c0eb2e4224.png">

That all makes sense, though I am having trouble finding a suitable prior distribution with which to update. `NormalInverseGamma` does not seem to be available, but I do see `Distributions.NormalInverseGaussian`. Would that be the correct prior in this case?

Reply:
There's no normal-inverse gamma distribution in Distributions IIRC. However, I don't think you need it either: In the model it's factorized into the distribution of sigma^2 (first line of the model) and distribution of mu conditioned on sigma^2 (second line) (you can compare it with https://en.wikipedia.org/wiki/Normal-inverse-gamma_distribution#Definition, in our example we use N-IG(mu=0, lambda=1, alpha=2, beta=3)). For checking the MCMC approximations of the posterior, for a quick check I guess the mean of of the updated N-IG distribution might be sufficient (given e.g. in the summary box on Wikipedia), that's also what we usually check in the Turing.jl tests. Alternatively, you could use that the marginal distributions are simply an IG distribution (for the variance) and a student t distribution (for the mean): https://en.wikipedia.org/wiki/Normal-inverse-gamma_distribution#Marginal_distributions

Reply:
Thanks for the helpful pointers - PR is updated